test/test_infra.py

```diff
@@ -103,12 +103,13 @@ def test_ensure_copyright(self):
             self.fail("Please add copyright headers to the following files:\n" + "\n".join(issues))
 
     def _run_setup(self, *args):
+        # from 3.6 we could just use encoding="utf8" here, but 3.5
+        # doesn't expose that and instead defaults to the wrong thing
         proc = subprocess.run(
             (sys.executable, 'setup.py') + args,
             stdout=subprocess.PIPE,
-            universal_newlines=True,
             check=True)
-        return proc.stdout.strip()
+        return proc.stdout.strip().replace(b"\r\n", b"\n").decode("utf8")
```

Comment:
Wondering why this is not `.replace(os.linesep, b"\n")` ? For instance you could just have "\r" as line separator if I am not mistaken.